lib/tasks/forms.rake

```diff
@@ -7,4 +7,13 @@ namespace :forms do
 
     puts "external_id has been set for each form to their id"
   end
+
+  desc "Set submission_type to email_with_csv"
+  task :set_submission_type_to_email_with_csv, %i[form_id] => :environment do |_, args|
```

Comment:
Optional: you could put a usage message in here (like the one in access_tokens.rake) to make dealing with errors easier

Reply:
Great suggestion, we'll do that

Reply:
done